root/account/applications/RevokeApplicationAccess.js

```diff
@@ -8,18 +8,32 @@
  */
 
 import ConfirmLayout from '../../components/ConfirmLayout.js';
+import {ACCESS_SCOPE_PERMISSIONS} from '../../constants.js';
+import {commaOnlyListText}
+  from '../../static/scripts/common/i18n/commaOnlyList.js';
 
 type Props = {
+  +application: ApplicationT,
   +form: SecureConfirmFormT,
+  +permissions: $ReadOnlyArray<number>,
 };
 
 const RevokeApplicationAccess = ({
+  application,
   form,
+  permissions,
 }: Props): React$Element<typeof ConfirmLayout> => (
   <ConfirmLayout
     form={form}
-    question={l(
-      `Are you sure you want to revoke this application's access?`,
+    question={texp.l(
+      `You’re about to revoke {app}’s permissions to: {permissions}.
+       Are you sure you want to revoke this application's access?`,
```

Comment:
The first (new) sentence is using typographic apostrophe, so the second one should now use the same.
```suggestion
       Are you sure you want to revoke this application’s access?`,
```

Reply:
I thought we were not changing apostrophes when it'd break translations?

Reply:
Oh, wait, I'm breaking it anyway - although with your new suggestion maybe I could not do so, I'll check.